conf/application.conf

```diff
@@ -60,4 +60,4 @@ tracking-consent-frontend {
   gtm.container = "d"
 }
 
-contact-frontend.serviceId = "Service Availability"
\ No newline at end of file
+contact-frontend.serviceId = "customs-service-status-frontend"
```

Comment:
minor: i think you can specify `${appName}` rather than hardcoding the name again here

Reply:
🙏 @sureshhmrc